src/utils/generator.js

```diff
@@ -192,6 +192,18 @@ export class Generator {
       nodes => t.callExpression(createCallee(), nodes));
   }
 
+  emitVoidExpression(types: TypesDomain, values: ValuesDomain, args: Array<Value>, buildNode_: AbstractValueBuildNodeFunction | BabelNodeExpression): UndefinedValue {
+    this.body.push({
+      args,
+      buildNode: (nodes: Array<BabelNodeExpression>) => t.expressionStatement(
+        (buildNode_: any) instanceof Function ?
```

Comment:
This ternary expression seems to be a common pattern. Extract to a shared helper method?